config/prod/test-ahayden-rds-postgres.yaml

```diff
@@ -0,0 +1,43 @@
+# Provision an EC2 instance
+template_path: remote/managed-rds-postgres-v1.j2
+stack_name: test-ahayden-rds-postgres
+sceptre_user_data:
+  # (Optional) Expose database ports to this whitelist of CIDR blocks
+   OpenIngress: ["207.109.76.122"]
```

Comment:
Should it not be gated by the VPN?

Reply:
Right? Nope, this is layer three whitelisted to an app on the web.